proto/substrait/algebra.proto

```diff
@@ -891,6 +911,12 @@ message Expression {
     }
   }
 
+  enum WindowFrame {
+    WINDOW_FRAME_UNSPECIFIED = 0;
+    WINDOW_FRAME_ROWS = 1;
+    WINDOW_FRAME_RANGE = 2;
+  }
```

Comment:
How does this define a window frame?  How many rows?  How big is the range?

For example, given the query:

`SELECT max(col) OVER (ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING) FROM my_table`

It's not clear where the `1 PRECEDING` and `1 FOLLOWING` would go.

Also, I don't see any support for `CURRENT ROW` or `UNBOUNDED`.

Reply:
The [Bound ](https://github.com/substrait-io/substrait/blob/9c350c953c4901aba4525fa6f198c98cb8fc4590/proto/substrait/algebra.proto#L850) already define the specification about which row to start  and where to end it in the Window.

Reply:
I missed that.  Thank you for pointing it out.